torch/_meta_registrations.py

```diff
@@ -1635,6 +1635,29 @@ def upsample_nearest2d_vec(input, output_size, scale_factors):
         memory_format=mem_format
     )
 
+def check_argmax_argmin(name, self, dim):
+    if dim is not None:
+        dim = maybe_wrap_dim(dim, self.dim())
+        zero_numel_check_dims(self, dim, name)
+    else:
+        check(
+            self.numel() != 0,
+            lambda: f"{name}: Expected reduction dim to be specified for input.numel() == 0.",
+        )
+
+
+@register_meta(aten.argmax.default)
+def argmax_meta(self, dim=None, keepdim=False):
+    check_argmax_argmin("argmax", self, dim)
+    dims = utils.reduction_dims(self.shape, (dim,) if dim is not None else None)
+    shape = _compute_reduction_shape(self, dims, keepdim)
+    return self.new_empty(shape, dtype=torch.int64)
+
+@register_meta(aten.scalar_tensor.default)
+def scalar_tensor(s, dtype=None, layout=None, device=None, pin_memory=None):
+    return torch.empty(
+        (), dtype=dtype, layout=layout, device=device, pin_memory=pin_memory
+    )
```

Comment:
testing?

Reply:
It's not clear this is correct, because I think `scalar_tensor` needs to infer dtype from s when dtype is not explicitly specified

Reply:
not sure, you'll have to check

Reply:
ok ya the tests run now
```
test_make_fx_exhaustive_scalar_tensor_cpu_float32 (__main__.TestProxyTensorOpInfoCPU) ... ok
test_make_fx_fake_exhaustive_scalar_tensor_cpu_float32 (__main__.TestProxyTensorOpInfoCPU) ... ok
test_make_fx_symbolic_exhaustive_inplace_scalar_tensor_cpu_float32 (__main__.TestProxyTensorOpInfoCPU) ... skipped 'No inplace variable for this op'
test_make_fx_symbolic_exhaustive_scalar_tensor_cpu_float32 (__main__.TestProxyTensorOpInfoCPU) ... ok
```